### ISSUE TITLE
Tab order broken after selecting date

### ISSUE THREAD

**Issue by cbebry** [Pikaday/Pikaday#346]
Setup more than one input field in a form, at least one of them being a pikaday input.
Focus on the pikaday input by tabbing into or clicking the field, then pick a date.
Press tab. The next field is not focused, instead (in Chrome at least) the first tab-able element on the page is selected instead.

I believe this has to do with the resolution of the following issue:
https://github.com/dbushell/Pikaday/issues/94

Since the field is blurred after the user selects the date, there is no "next field" to focus on in the tab order. This causes a user experience issue, as the user expects to be able to tab through while filling out every piece of the form.

Similar, possibly related issue:
https://github.com/dbushell/Pikaday/issues/235

Thank you for your time.

**Comment by nathancahill** [comment]
Interesting. I can only replicate this in Chrome. Firefox does the "right thing" in tabbing to the next field.

**Comment by cbebry** [comment]
That really is interesting. I tried to search for a chromium report on the issue but have not found one yet.

Would it be possible to make the blur a configurable option if some others need that functionality like in #94 ? This way for my particular use case I could choose to turn it off.

**Comment by cbebry** [comment]
I tested IE 9-11 as well, and all of those versions can replicate this bug.

**Comment by nathancahill** [comment]
I see. Well, the field can always be blurred with the `onClose` callback. So I think the correct thing to do here is to not blur it in the library (as opposed to having it as an option).

**Comment by cbebry** [comment]
I am OK with this.

**Comment by akc42** [comment]
Did anything more come of this?  I am now experiencing the same issue.  I tried the fix in #346, but that didn't seem to solve it. 

What might be different in my case is I have had to provide a fake input field, which is set with type="hidden", disabled and tabindex="-1" as I don't want pikaday uncontrollably writing to my real input.

**Comment by akc42** [comment]
I eventually figured out what is happening.  Pikaday creates the picker with button elements but without tabindex="-1" on them.  When the bound field blurs, tab jumps to the pikaday button element in the picker which is then immediately hidden.  The active element is then left somewhere else (in my case body).

I hacked a solution together in my private copy of making the picker buttons all have tabindex="-1" on them.  At the very least this might be a useful option, although it is probably the case that whenever there is a bound field specified then this should apply.
